### ISSUE TITLE
Does not import abf

### ISSUE THREAD

**Issue by abuchin** [NeuralEnsemble/python-neo#380]
Hello dear users,

I have recently found the following bug with neo files. When I try to import the abf file, it could not read it. I use the following commands:
```
r = io.AxonIO('My_file.abf')
bl = r.read()

Instead of getting results I get the following error:

AttributeError                            Traceback (most recent call last)
<ipython-input-33-41a0322d6322> in <module>()
      7 
      8 r = io.AxonIO(filename=abf_file_list[i])
----> 9 bl = r.read()
     10 
     11 bl

/Users/Anatoly/anaconda/lib/python2.7/site-packages/neo/io/baseio.pyc in read(self, lazy, cascade, **kargs)
    114                 return self.read_all_blocks(lazy=lazy, cascade=cascade,
    115                                             **kargs)
--> 116             return [self.read_block(lazy=lazy, cascade=cascade, **kargs)]
    117         elif Segment in self.readable_objects:
    118             bl = Block(name='One segment only')

/Users/Anatoly/anaconda/lib/python2.7/site-packages/neo/io/axonio.pyc in read_block(self, lazy, cascade)
    318                 for i, tag in enumerate(header['listTag']):
    319                     times.append(tag['lTagTime']/sampling_rate)
--> 320                     labels.append(tag['nTagType'].decode("utf-8"))
    321                     comments.append(clean_string(tag['sComment']))
    322                 times = np.array(times)

AttributeError: 'int' object has no attribute 'decode'
```
I am running Python 2.7 Anaconda on Mac OS 10.12.6 Sierra, macbook pro. Any help would be appreciated. Surprising the same code works well for the other abf files, but not for all of them.

Looking forward for your help.
Anatoly

**Comment by samuelgarcia** [comment]
In the master line 320 is:
```
labels.append(str(tag['nTagType']))
```

This have been corrected by cjfraz here
https://github.com/NeuralEnsemble/python-neo/commit/860d978dad46029bd34ac30fae791b386ecc67cb#diff-28dbfa72cda9619fb76c682565ec2e51


Could you clone the master branch and install it for testing ?

**Comment by abuchin** [comment]
Thanks, it helped! Now I could upload the abf files, since I have hunderts of them it is almost impossible to do the conversion manually.

Is it he part of the current master branch? Could this improvement of neo package already in 0.5.1 release?

**Comment by samuelgarcia** [comment]
This will be included in  0.5.2 hopefully soon.
